{{ cookiecutter.__project_name_kebab }}/tox.ini

```diff
@@ -3,15 +3,17 @@ skipsdist = True
 usedevelop = True
 
 envlist =
-    python{3.8,3.9,3.10,3.11}-django{3.2,4.0,4.1}-wagtail{4.1,4.2}-{sqlite,postgres}
-    python{3.8,3.9,3.10,3.11}-django{3.2,4.1,4.2}-wagtail{5.0,5.1,main}-{sqlite,postgres}
+    python{3.8,3.9,3.10,3.11}-django{3.2}-wagtail{4.1}-{sqlite,postgres}
+    python{3.8,3.9,3.10,3.11}-django{3.2,4.2}-wagtail{5.1,5.2,main}-{sqlite,postgres}
+    python{3.12}-django{4.2}-wagtail{5.2,main}-{sqlite,postgres}
```

Comment:
we normally keep `wagtailmain` out of the test matrix, deferring to the nightly CI task instead. This is so that we don't get failures against it in the normal course of operations. 
Alas, GitHub Actions doesn't have a nice "allow-failures" mode (ref https://github.com/actions/runner/issues/2347)

Reply:
if you want to allow tox to run tests against `wagtailmain` locally, you could add something like

```ini
[testenv:wagtailmain]
deps =
   git+https://github.com/wagtail/wagtail.git
```

Reply:
Just to be clear, `wagtailmain` is not a new target I introduced in this PR

https://github.com/wagtail/cookiecutter-wagtail-package/blob/d9ccc7e7204a0e64c562e070af8113f949720c80/%7B%7B%20cookiecutter.__project_name_kebab%20%7D%7D/tox.ini#L7

I've dropped it in https://github.com/wagtail/cookiecutter-wagtail-package/pull/70/commits/9b882ba34730d717190d77839abd33aeed314bfb anyway

Reply:
😱